eusurdf/euslisp/eusmodel_to_urdf.l

```diff
@@ -0,0 +1,62 @@
+;; eusmodel_to_urdf.l
+;; Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
+
+;; assumed executed at PROJECT_SOURCE_DIR
+(defvar *eusurdf-package-dir* (unix:getwd))
+(require :conversion-utils (format nil "~A/euslisp/conversion-utils.l" *eusurdf-package-dir*))
+
+(defun make-model-conf (name fpath &optional desc)
+  (unless desc
+    (setq desc (format nil "This is automatically generated eus model from ~A" name)))
+  (with-open-file (f fpath :direction :output :if-exists :overwrite)
+    (format f "<?xml version='1.0'?>~%")
+    (format f "<model>~%")
+    (format f "  <name>~A</name>~%" name)
+    (format f "  <version>0.1.0</version>~%")
+    (format f "  <sdf>model.urdf</sdf>~%")
+    (format f "  <description>~%")
+    (format f "    ~A~%" desc)
+    (format f "  </description>~%")
+    (format f "</model>~%")))
+
+(defun eusmodelfile2urdf (fname outdir &key (tmpdir "/tmp/") collada-to-urdf-exe-path)
+  (unless (probe-file fname)
+    (errorf "file ~A not exists" fname))
+  (load fname)
+  (let* ((model-name (string-join "-" (butlast (string-split (send (pathname fname) :name) #\-))))
+         (model (funcall (read-from-string model-name))))
+    (eusmodel2urdf model model-name outdir :tmpdir tmpdir :collada-to-urdf-exe-path collada-to-urdf-exe-path)))
+
+(defun eusmodel2urdf (model model-name outdir &key (tmpdir "/tmp/") collada-to-urdf-exe-path)
+  (let ((urdf-out-path (send (concatenate-pathnames outdir "model.urdf") :namestring))
+        (mesh-out-dir (send (concatenate-pathnames outdir "meshes/") :namestring)))
+    (cond ((null (send model :name)) (send model :name model-name))
+          ((symbolp (send model :name)) (send model :name (string-downcase (send model :name)))))
+    (collada::eus2collada model tmpdir)
+    (make-dirs mesh-out-dir)
+    (unless collada-to-urdf-exe-path
+      (setq collada-to-urdf-exe-path "rosrun collada_urdf_jsk_patch collada_to_urdf"))
+    (unix:system (format nil "~A ~A -G -A --mesh_output_dir ~A --mesh_prefix \"model://~A/meshes\" -O ~A"
+                         collada-to-urdf-exe-path
+                         (send (concatenate-pathnames tmpdir model-name ".dae") :namestring)
+                         mesh-out-dir
+                         model-name
+                         urdf-out-path))
+    (unix:system (format nil "rm -f ~A"
```

Comment:
> Sometimes user does not have permission to remove files in /tmp directory , which other users generated.
Please see #124 again.

Above my comment was not correct. You remove collada file right after generating that. In that case, user who generate the file and remove file is same, so no problem. (sometimes we want to see collada file to check that conversion is executed correctly, and if collada is removed, we can't check that, but it is trivial thing, so OK.)

> I think tmp-dir-no-duplication option for (irteus2urdf-room-for-gazebo) is not necessary and it is ok to always generate no duplication directory: https://github.com/jsk-ros-pkg/jsk_model_tools/pull/188/files#diff-fd3566e2a31868b72476ad66c9e5e01dR55 .
 So please remove following section and move it to (eusmodel2urdf) : d6812fe#diff-fd3566e2a31868b72476ad66c9e5e01dR72

Now, we no longer generate no-duplicate directory as tempral directory, so could you remove  tmp-dir-no-duplication option and https://github.com/jsk-ros-pkg/jsk_model_tools/pull/188/files#diff-fd3566e2a31868b72476ad66c9e5e01dR72 section from (irteus2urdf-room-for-gazebo)?